### PR TITLE
Add sharable dev badge page

### LINKED ISSUE #959: Create a new page for sharable developer badges
**Is your feature request related to a problem? Please describe.**
We need a new page to be shared with the developers who achieve certain dev badges.

**Describe the solution you'd like**
- Create a new page under `/developers` with two URL parameters to get the username and the badge. 
- Set a fitting preview meta image for the link to be shown when shared on social media platforms

### PR DESCRIPTION
**Pull request by Gravewalker666**
## Purpose

The purpose of this PR is to fix #959 

## Goals

- To create a new page to share among sef developers with their badges in it

## Approach

- Made the url to require two parameters for `user` and `badge`
- Took the user profile from sef hive
- Rendered them out using mustache

### Screenshots

image

  
### Preview Link

https://pr-960-sef-site.surge.sh/developers?user=jaye&badge=5
https://pr-960-sef-site.surge.sh/developers?user=Gravewalker&badge=3

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [x] I have read and understood the development best practices guidelines ( http://bit.ly/sef-best-practices )
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

## Test environment
- Safari
- MacOs Big Sur

### CODE REVIEW

File: developers/index.html [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        gtag('config', 'UA-167873271-1');
+    </script>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <meta name="description" content="ScholarX is an exclusive 6-month program aimed at providing
+                                      mentoring support to a selected pool of high Potential undergraduate students
+                                      based in Sri Lanka ideally by a Sri Lankan expat currently engaged with one of
+                                      the world’s top universities or Fortune 500 companies. It’s our free premium
+                                      mentoring platform by Sri Lankans for Sri Lankans working towards creating a culture
+                                      of knowledge and expertise sharing without the limitation of geographical borders.">
```
Comment (anjula-sack) [inline developers/index.html]: meta data needs to be chaned right?

**Review by anjula-sack — CHANGES_REQUESTED** [review]
This is amazing! @Gravewalker666 😍😍😍

File: developers/index.html (hunk 1 above)
Comment (Gravewalker666) [reply developers/index.html]: Oh shit, I forgot, I'll update it 
Thank you for pointing it out @anjula-sack

Comment (Gravewalker666) [reply developers/index.html]: Resolved with the commit 6cc6fe1
Please review @anjula-sack

**Review by anjula-sack — APPROVED** [review]
Amazing work! @Gravewalker666

**Pull request merged**